LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Cameron Sparr
+Copyright (c) 2016–2020 Cameron Sparr and contributors
```

Comment:
It's unnecessary to change the date. This just creates toil of having to update this date every year.

Reply:
Good point. How's this instead?

```suggestion
Copyright (c) 2016 and later, Cameron Sparr and contributors.
```

Reply:
It can just be the original date, iirc, "and later" is implied.

Reply:
Done! 🆗